src/main/java/com/sngular/kloadgen/sampler/schemaregistry/adapter/impl/ParsedSchemaAdapter.java

```diff
@@ -0,0 +1,7 @@
+package com.sngular.kloadgen.sampler.schemaregistry.adapter.impl;
+
+public abstract class ParsedSchemaAdapter {
```

Comment:
This abstract class only defines function signatures, should it be an interface instead?